.travis.yml

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright 2017 HyperHQ Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+sudo: required
+dist: trusty
+
+language: go
+go_import_path: github.com/kata-containers/agent
+
+go:
+  - 1.8
```

Comment:
How about also adding the following?:

```
  - 1.9
  - tip
```

Reply:
I'm OK with both. Just note that other components are also using v1.8. If we want to go with v1.9 and tip, we should change others' travis settings as well.

Reply:
Ah - sorry, I forgot to check the other components. I'm happy to raise PRs to add these 2 versions in for the others if we all agree to use `1.9` and `tip` though?

Reply:
yeah, sounds good to me!